modules/ROOT/pages/clauses/limit.adoc

```diff
@@ -141,7 +151,50 @@ Writes `locked` property on one node and return that node:
 [role="queryresult",options="header,footer",cols="1*<m"]
 |===
 | n
-| {locked:true,name:"A",age:60}
+| {locked:true,name:"Andy",age:60}
 d|Rows: 1 +
 Properties set: 1
 |===
+
+[role=label--new-5.24]
+[[limit-standalone-clause]]
+== Using `LIMIT` as a standalone clause
+
+`LIMIT` can be used as a standalone clause, or in conjunction with xref:clauses/order-by.adoc[`ORDER BY`] or xref:clauses/skip.adoc[`SKIP`]/xref:clauses/skip.adoc#offset-synonym[`OFFSET`].
+
+.Standalone use of `LIMIT`
+[source, cypher]
+----
+MATCH (n)
+LIMIT 2
+RETURN collect(n.name) AS names
+----
+
+.Result
+[role="queryresult",options="header,footer",cols="1*<m"]
+|===
+| names
+| ["Andy", "Bernard"]
+|Rows: 1
+|===
```

Comment:
@JPryce-Aklundh Here I think we should clarify that the ordering is not guaranteed, so users do not believe they will get ascending ordering by default.

Reply:
We seem to have this disclaimer for SKIP which I believe is applicable here as well. 

_Neo4j does not guarantee the results generated by SKIP/OFFSET. The only clause that guarantees a specific row order is [ORDER BY](https://neo4j-docs-cypher-1027.surge.sh/cypher-manual/5/clauses/order-by/)._